libcontainer/container_userns_linux.go

```diff
@@ -21,6 +24,15 @@ func (c *linuxContainer) addUidGidMappings(sys *syscall.SysProcAttr) error {
 			sys.GidMappings[i].HostID = gm.HostID
 			sys.GidMappings[i].Size = gm.Size
 		}
+
+		// Set sys.GidMappingsEnableSetgroups to true, so the process's
```

Comment:
I agree a build tag is cleaner than reflection, but wanted to avoid adding another file or even two (depending on whether we'd like to duplicate addUidGidMappings or add another function).

If you'd still like to do this with build tags, please let me know.